Appendix: Weird categories (#18)
MrLongbottom/P10-Article: sections/appendix/categories.tex

\subsection{Metadata labels}\label{sec:appendix_meta_data}
Here are the 58 different categories, which make up all the category labels present within the dataset.

> As mentioned in \autoref{sec:dataset}, there are some nonsensical categories that are worth mentioning.

Reviewer: ```suggestion
As mentioned in \autoref{sec:dataset}, some nonsensical categories are worth mentioning.
```

Specifically, the categories '26. Frederik' and '53. Frederik' do not from their names indicate what they cover, since Frederik is simply a normal Danish name.
Since they are not filtered into the 'misc' category during preprocessing, it is worth looking into what documents have these categories.

> By looking at a random selection of documents from these categories, there can not be seen any pattern of topics.

Reviewer: ```suggestion
By looking at a random selection of documents from these categories, there is no clear pattern to be found.
```

> The topics written can be about anything from sports to news from around the world or locally in Denmark.

Reviewer: ```suggestion
The topics, within these categories, can be about anything from sports to news from anywhere around the world or locally in Denmark.
```
er det om kategoriene eller generelt?

> All articles seem to be from 2019, the last year in our dataset, but spread over the whole year.

Reviewer: ```suggestion
The articles from '26. Frederik' appear all years in our dataset, while the articles for '53. Frederik' seem to be just from 2019, the last year in our dataset, but evenly distributed over the whole year.
```
er det sådan nogenlunde ?

> Curiously enough, most of the documents from these categories seem to be written by Anders Kjærgaard, with only a few other authors.

Reviewer: ```suggestion
Curiously enough, most of the documents from these categories seem to be written by Anders Kjærgaard, with only a few other interfering authors.
```

For '26. Frederik' there are just 5 unique authors: Carsten Tolbøll, Emil Abkjær Kristensen, Morten Kyndby Holm, Jens Fogh-Andersen, and Anders Kjærgaard.
For '53. Frederik' there are just 4 unique authors: Klaus Færch Gjerulff, Morten Kyndby Holm, Jens Fogh-Andersen, and Anders Kjærgaard.
This shows that Morten Kyndby Holm, Jens Fogh-Andersen, and Anders Kjærgaard have written for both categories.

From the exploration of these two categories, we can not say with certainty why these categories exist, or why multiple authors have chosen to write for these categories.

> We continue to use these categories in our experiment, because we might make other observations through the topic models.

Reviewer 2: ```suggestion
We continue to use these categories in our experiment, for the possibility of making other observations through the topic models.
```
